readme.md

```diff
@@ -196,7 +194,7 @@ In v22.2, use the following CSS rules:
 [Return to the table of contents.](#thetableofcontents)
 
 
-### Change the header cells' background color
+### Change the Header Cells' Background Color
 
 In v22.1, use the following CSS rules:
```

Comment:
In the context of the CSS below, is "CSS rules" (plural) correct? Or should it be "CSS rule" (singular)? Just checking.

Reply:
I checked and corrected if necessary